.github/workflows/docs.yml

```diff
@@ -10,10 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: ['11']
+        java: ['17']
```

Comment:
Do we really need a matrix here?

Reply:
I've made all of these github workflow changes.